src/main/java/xyz/nucleoid/extras/sidebar/NucleoidSidebar.java

```diff
@@ -101,47 +99,47 @@ public void update(long ticks, MinecraftServer server, NucleoidExtrasConfig conf
             b.add(SIZE_FORCING_TEXT);
             b.add((p) -> {
                 if (p != null) {
-                    return new LiteralText("» ").append(
-                            new TranslatableText("nucleoid.sidebar.welcome",
-                                    new LiteralText("").formatted(Formatting.WHITE).append(p.getDisplayName())
+                    return Text.literal("» ").append(
+                            Text.translatable("nucleoid.sidebar.welcome",
+                                    Text.literal("").formatted(Formatting.WHITE).append(p.getDisplayName())
                             ).setStyle(TOP_SIDEBAR_STYLE)
                     ).formatted(Formatting.GRAY);
                 } else {
-                    return LiteralText.EMPTY;
+                    return Text.empty();
```

Comment:
Not important whatsoever, but random side-note: for non-mutable texts, `ScreenTexts.EMPTY` can be used (unfortunate Yarn naming there 😅)